r2dbc-mysql/src/main/java/SimpleStatement.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.jasync.r2dbc.mysql
+
+import io.r2dbc.spi.Result;
+import io.r2dbc.spi.Statement;
+import io.reactivex.BackpressureStrategy
+import io.reactivex.Flowable
+import org.reactivestreams.Publisher
+import java.util.function.Supplier
+import com.github.jasync.sql.db.Connection as JasyncConnection
+
+class SimpleStatement(private val clientSupplier: Supplier<JasyncConnection>, private val sql: String) : Statement {
+
+    override fun add(): Statement {
+        TODO("not implemented")
+    }
+
+    override fun bind(identifier: Any, value: Any): Statement {
+        TODO("not implemented")
+    }
+
+    override fun bind(index: Int, value: Any): Statement {
+        TODO("not implemented")
+    }
+
+    override fun bindNull(identifier: Any, type: Class<*>): Statement {
+        TODO("not implemented")
+    }
+
+    override fun bindNull(index: Int, type: Class<*>): Statement {
+        TODO("not implemented")
+    }
+
+    override fun execute(): Publisher<out Result> {
+        return Flowable.create({ emitter ->
+
+            val jasyncConnection = clientSupplier.get();
+
+            jasyncConnection.sendQuery(this.sql).handle { a, t: Throwable? ->
+
+                if (t == null) {
+
+                    val result = a.rows
+                    result.forEach {
+                        emitter.onNext(JaysncResult(result));
```

Comment:
This line is not clear to me. why are we sending the result more than once?

Reply:
@mp911de - if you can help with that it will be great, as well as general review. I didn't add transactions support yet.

Reply:
A statement can contain two SQL operations (such as `SELECT * FROM foo; SELECT * FROM bar` or `INSERT INTO foo VALUES(…); UPDATE bar SET foo = 'bar';…`). Therefore, a database can return more than one result.

Reply:
Got it. Our driver does not support multiple result set, so I will change that.